libcxx/test/std/algorithms/ranges_robust_against_dangling.pass.cpp

```diff
@@ -13,7 +13,7 @@
 // Range algorithms should return `std::ranges::dangling` when given a dangling range.
 
```

Comment:
The formatting changes in here are nice, but would be better handled in a separate PR, just to make it clear what actually changed.

Reply:
I agree and apologize for accidentally formatting this, it does make the diff harder to read. Do you want me to revert the formatting here or just keep this in mind in the future?

Reply:
Please revert and apply in a separate patch.

Reply:
This should be resolved with [be7faa6](https://github.com/llvm/llvm-project/pull/68494/commits/be7faa6fb11c09822a9a855c7cf1aecab4f4c773)